renderer/components/google-calendar/GoogleCalendarAddEventBtn.tsx

```diff
@@ -68,6 +89,31 @@ export default function GoogleCalendarAddEventBtn({ addEvent }) {
         event.from = from;
         event.to = to;
 
+        const items = summary ? summary.split(" - ") : "";
+
+        switch (items.length) {
+          case 0:
+            event.description = "";
+            break;
+          case 1:
+            event.description = items[0];
+            break;
+          case 2:
+            if (availableProjects.includes(items[0])) {
+              event.project = items[0];
+              event.description = items[1];
+            } else {
+              event.activity = items[0];
+              event.description = items[1];
+            }
+            break;
+          case 3:
+            event.project = items[0];
+            event.activity = items[1];
+            event.description = items[2];
+            break;
+        }
+
```

Comment:
As I know, you must always add the default value in switch-case constuction

And I would add a case when `items.length` > 3. But it's not extremely needed